### PR TITLE
SDK Docs

### PR DESCRIPTION
**Pull request by AaronCrawfis**
# Description

- Renamed samples to examples to be consistent with all other SDKs
- Update home page README
- Add daprdocs repo with all content to be rendered in docs.dapr.io
- Split client examples into sub-directories

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:
* [x] Extended the documentation

### CODE REVIEW

**Comment by AaronCrawfis** [comment]
@rynowak @vinayada1 @msfussell there are 3 areas I haven't completed on this and need input:

- daprdocs\content\en\dotnet-sdk-docs\dotnet-client.md needs examples for secrets and bindings
- daprdocs\content\en\dotnet-sdk-docs\dotnet-sdk-extensions\dotnet-aspnet.md needs content
- daprdocs\content\en\dotnet-sdk-docs\dotnet-sdk-extensions\dotnet-configuration.md needs content

**Comment by AaronCrawfis** [comment]
@rynowak @vinayada1 @msfussell there are 3 areas I haven't completed on this and need input:

- daprdocs\content\en\dotnet-sdk-docs\dotnet-client.md needs examples for secrets and bindings
- daprdocs\content\en\dotnet-sdk-docs\dotnet-sdk-extensions\dotnet-aspnet.md needs content
- daprdocs\content\en\dotnet-sdk-docs\dotnet-sdk-extensions\dotnet-configuration.md needs conent

File: daprdocs/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+This page covers how the documentation is structured for the Dapr .NET SDK.
+
+## Dapr Docs
+
+All Dapr documentation is hosted at [docs.dapr.io](https://docs.dapr.io), including the docs for the [.NET SDK](https://docs.dapr.io/developing-applications/sdks/dotnet/). Head over there if you want to read the docs.
+
+### .NET SDK docs source 
+
+Although the docs site code and content is in the [docs repo](https://github.com/dapr/docs), the .NET Python SDK content and images are within the `content` and `static` directories, respectively.
```
Comment (rynowak) [inline daprdocs/README.md:11]: Surprisingly `.NET Python` was once a thing: https://ironpython.net/

File: daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+linkTitle: ".NET SDK"
+weight: 3000
+description: Guidelines for contributing to the Dapr .NET SDK
+---
+
+When contributing to the [.NET SDK](https://github.com/dapr/dotnet-sdk) the following rules and best-practices should be followed.
+
+## Samples
+
+The `samples` directory contains code samples for users to run to try out specific functionality of the various .NE SDK packages and extensions. When writing new and updated samples keep in mind:
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md:13]:
```suggestion
The `samples` directory contains code samples for users to run to try out specific functionality of the various .NET SDK packages and extensions. When writing new and updated samples keep in mind:
```

File: daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- All examples should be runnable on Windows, Linux, and MacOS. While Python code is consistent among operating systems, any pre/post example commands should provide options through [codetabs]({{< ref "contributing-docs.md#tabbed-content" >}})
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-contributing/dotnet-contributing.md:15]:
```suggestion
- All examples should be runnable on Windows, Linux, and MacOS. While .NET Core code is consistent among operating systems, any pre/post example commands should provide options through [codetabs]({{< ref "contributing-docs.md#tabbed-content" >}})
```

File: daprdocs/content/en/dotnet-sdk-docs/_index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Dapr .NET SDK"
+linkTitle: ".NET"
+weight: 1000
+description: .NET SDK packages for developing Dapr applications
+no_list: true
+---
+
+Dapr offers a variety of packages to help with the development of Python applications. Using them you can create .NE clients, servers, and virtual actors with Dapr.
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-docs/_index.md:10]:
```suggestion
Dapr offers a variety of packages to help with the development of .NET applications. Using them you can create .NET clients, servers, and virtual actors with Dapr.
```

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-actor.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Getting started with the Dapr actor .NET SDK"
+linkTitle: "Actor"
+weight: 20000
+description: How to get up and running with the Dapr .NET SDK
+---
+
+The Dapr actor package allows you to interact with Dapr virtual actors from a Python application.
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-docs/dotnet-actor.md:9]:
```suggestion
The Dapr actor package allows you to interact with Dapr virtual actors from a .NET application.
```

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-actor.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [.NET 5.0+](https://dotnet.microsoft.com/download) installed
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-docs/dotnet-actor.md:15]:
```suggestion
- [.NET 3.1+](https://dotnet.microsoft.com/download) installed
```

.NET 5.X is required to build **this** repo. 3.1 or newer is required for building applications with Dapr.

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actor-howto.md [hunk 7]
```diff
@@ -1,9 +1,16 @@
...
+linkTitle: "Example"
+weight: 100000
+description: Try out .NET virtual actors with this example
+---
+
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [.NET 3.1+](https://dotnet.microsoft.com/download) installed
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-docs/dotnet-actors/dotnet-actor-howto.md:13]: This is confusing, but needs to be clarified. The right branding for 3.1 is `.NET Core 3.1` - the right branding for 5.0 is `.NET 5+`

```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+no_list: true
+---
+
+The Dapr client package allows you to interact with other Dapr applications from a .NET application.
+
+## Prerequisites
+
+- [Dapr CLI]({{< ref install-dapr-cli.md >}}) installed
+- Initialized [Dapr environment]({{< ref install-dapr-selfhost.md >}})
+- [.NET 5.0+](https://dotnet.microsoft.com/download) installed
```
Comment (rynowak) [inline daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md:16]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/Actor/README.md [hunk 9]
```diff
@@ -1,32 +1,23 @@
...
-The Actor sample shows how to create an Actor (`DemoActor`) and invoke its methods on the client application.
+The Actor example shows how to create a virtual actor (`DemoActor`) and invoke its methods on the client application.
 
-## Prerequistes
+## Prerequisites
 
-* [.Net Core SDK 3.0](https://dotnet.microsoft.com/download)
-* [Dapr CLI](https://github.com/dapr/cli)
-* [Dapr DotNet SDK](https://github.com/dapr/dotnet-sdk)
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/Actor/README.md:7]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/AspNetCore/ControllerSample/README.md [hunk 10]
```diff
@@ -9,7 +9,14 @@ It exposes the following endpoints over HTTP:
 
 The application also registers for pub/sub with the `deposit` and `withdraw` topics.
 
- ## Running the Sample
+## Prerequisitess
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/AspNetCore/ControllerSample/README.md:14]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/AspNetCore/GrpcServiceSample/README.md [hunk 11]
```diff
@@ -9,7 +9,14 @@ It exposes the following endpoints over GRPC:
 
 The application also registers for pub/sub with the `deposit` and `withdraw` topics.
 
- ## Running the Sample
+## Prerequisitess
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/AspNetCore/GrpcServiceSample/README.md:14]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/AspNetCore/RoutingSample/README.md [hunk 12]
```diff
@@ -9,7 +9,14 @@ It exposes the following endpoints over HTTP:
 
 The application also registers for pub/sub with the `deposit` and `withdraw` topics.
 
- ## Running the Sample
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/AspNetCore/RoutingSample/README.md:14]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/AspNetCore/SecretStoreConfigurationProviderSample/README.md [hunk 13]
```diff
@@ -1,17 +1,19 @@
-# Dapr Secret Store Configuration Provider in ASP.NET Core
+# Dapr secret store configuration provider in ASP.NET Core
 
-## Prerequistes
-* [.Net Core SDK 3.1](https://dotnet.microsoft.com/download)
-* [Dapr CLI](https://github.com/dapr/cli)
-* [Dapr DotNet SDK](https://github.com/dapr/dotnet-sdk)
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/AspNetCore/SecretStoreConfigurationProviderSample/README.md:5]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/Client/PublishSubscribe/README.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
+# Dapr .NET SDK pub/sub example
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/Client/PublishSubscribe/README.md:5]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/Client/ServiceInvocation/README.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
+# Dapr .NET SDK service invocation example
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/Client/ServiceInvocation/README.md:5]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: examples/Client/StateManagement/README.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
+# Dapr .NET SDK state management example
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline examples/Client/StateManagement/README.md:5]:
```suggestion
- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

File: samples/Client/README.md [hunk 17]
```diff
@@ -1,12 +1,12 @@
...
+The following examples will show you how to:
 
 - Invoke services
 - Publish events
 - Use the state store to get, set, and delete data
 
-## Prerequistes
+## Prerequisites
 
 * [.Net Core SDK](https://dotnet.microsoft.com/download)
```
Comment (rynowak) [inline samples/Client/README.md:11]:
```suggestion
* [.Net Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download)
```

**Comment by rynowak** [comment]
@AaronCrawfis - can you resolve conflicts here? Also should this target release/1.0 instead?

**Review by rynowak — DISMISSED** [review]

**Review by rynowak — APPROVED** [review]

**Pull request merged**